frontend/pages/validator/_id/index.vue

```diff
@@ -214,22 +214,6 @@
               />
             </div>
           </div>
-          <div class="row">
```

Comment:
Why did you remove this?

Reply:
Removed this because this row is empty on the page.  `Gevernence` component was not imported in the initial version of UI so it wasn't rendered but caused a warming in the development mode.